packages/@sanity/form-builder/src/inputs/files/ImageToolInput/ImageToolInput.tsx

```diff
@@ -92,32 +89,19 @@ export default class ImageToolInput extends React.Component<
     return (
       // @todo: render presence and markers
       <ChangeIndicatorProvider
-        path={HOTSPOT_PATH}
-        focusPath={EMPTY_PATH}
+        path={['hotspot']}
+        focusPath={[]}
```

Comment:
Do these have special handling now that doesn't cause re-rendering?

Reply:
Good point, fixed to use never-changing constants.